packages/react/src/prefabs/PreJoin.tsx

```diff
@@ -362,41 +448,43 @@ export function PreJoin({
         )}
       </div>
       <div className="lk-button-group-container">
-        <div className="lk-button-group audio">
+        <div className="lk-button-group-pre-join audio">
           <TrackToggle
             initialState={audioEnabled}
             source={Track.Source.Microphone}
             onChange={(enabled) => setAudioEnabled(enabled)}
-          >
-            {micLabel}
-          </TrackToggle>
-          <div className="lk-button-group-menu">
-            <MediaDeviceMenu
-              initialSelection={audioDeviceId}
-              kind="audioinput"
-              disabled={!audioTrack}
-              tracks={{ audioinput: audioTrack }}
-              onActiveDeviceChange={(_, id) => setAudioDeviceId(id)}
-            />
+          />
+          <div className="lk-button-group-menu-pre-join">
+            <label className="lk-selected-device-label">
+              {selectedAudioDevice?.label || micLabel}
```

Comment:
this is a nice idea, but the handling could be moved into the <MediaDeviceMenu /> instead in order to allow for the same experience everywhere (not only in the prejoin). 
Keeping it simple with a `useDeviceInfo(track)` hook that only returns the MediaDeviceInfo for that use case seems more appropriate